src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -461,6 +461,10 @@ def test_enable_disable_features(self, resource_group):
                 "connectedk8s disable-features -n {name} -g {rg} --features cluster-connect --kube-config \
                 {kubeconfig} --kube-context {managed_cluster_name}-admin -y"
             )
+        with self.assertRaisesRegex(CLIError, "Disabling 'cluster-connect' feature is not allowed when \
+'custom-locations' feature is enabled."):
+            self.cmd('connectedk8s disable-features -n {name} -g {rg} --features cluster-connect --kube-config \
+                {kubeconfig} --kube-context {managed_cluster_name}-admin -y')
```

Comment:
Why a new test is added?

Reply:
This might be introduced when Git merge conflict. Fixed.